package.json

```diff
@@ -20,29 +20,29 @@
     "lint": "eslint ."
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
   },
   "license": "MIT",
   "dependencies": {
-    "async": "~0.9.0",
-    "commondir": "0.0.1",
-    "debug": "^2.0.0",
-    "lodash": "^4.17.5",
-    "semver": "^4.1.0",
-    "strong-globalize": "^2.6.2",
-    "toposort": "^0.2.10"
+    "async": "^2.6.2",
```

Comment:
IIRC, there are subtle but important changes in the way how individual callbacks are scheduled/executed between async versions 0.x and post-1.x.

Unless there is a know security vulnerability in `async@0.9.0`, we should not be upgrading this dependency.

Reply:
Reverted.